src/course/course.controller.ts

```
@@ -0,0 +1,87 @@
+  }
+
+  @Get(':id')
+  async findOne(@Param('id') id: string): Promise<Course> {
+    if (!isValidObjectId(id)) {
+      throw new BadRequestException('Invalid course ID format');
+    }
+    return this.courseService.findOne(id);
+  }
+
+  @Put(':id')
+  @UseInterceptors(FilesInterceptor('attachments', 10, {
```

ESLint

[error] Insert `⏎····`

(prettier/prettier)